llvm/include/llvm/IR/IntrinsicsAArch64.td

```diff
@@ -855,6 +855,9 @@ def int_aarch64_crc32x  : DefaultAttrsIntrinsic<[llvm_i32_ty], [llvm_i32_ty, llv
     [IntrNoMem]>;
 def int_aarch64_crc32cx : DefaultAttrsIntrinsic<[llvm_i32_ty], [llvm_i32_ty, llvm_i64_ty],
     [IntrNoMem]>;
+def int_aarch64_ubfx : DefaultAttrsIntrinsic<
```

Comment:
This should perhaps be not under the "CRC32" section.

Reply:
Addressed it in https://github.com/llvm/llvm-project/pull/80103/commits/affc65a205c95ed7f48daf80027c04b85debfe9a